api/helpers/vimeo.js

```diff
@@ -0,0 +1,74 @@
+const axios = require('axios');
+
+const EXTRACT_VIMEO_REGEX =
+  /(?:www\.|player\.)?vimeo.com\/(?:showcase\/(?:\w+\/)?|groups\/(?:[^\/]*)\/videos\/|album\/(?:\d+)\/video\/|video\/|)(\d+)(?:[a-zA-Z0-9_\-]+)?/i;
+
+const PrivateKey = process.env.VIMEO_PRIVATE_API_KEY;
+const PublicKey = process.env.VIMEO_PUBLIC_API_KEY;
+
+/**
+ * Extract Vimeo Video or Showcase ID from a URL
+ * @param {string} url Youtube URL
+ */
+const getId = (url) => {
+  const match = url.match(EXTRACT_VIMEO_REGEX);
+  if (match && match[1]) {
+    return match[1];
+  }
+  return null;
+};
+
+/**
+ * Get Youtube video duration
+ * @param {string} videoId Youtube Video ID
+ */
+const getSingleVideo = async (videoId) => {
+  try {
+    const url = `https://api.vimeo.com/videos/${videoId}`;
+    const {
+      data: { duration, link },
+    } = await axios.get(url, {
+      headers: {
+        Authorization: `Bearer ${PrivateKey}`,
+      },
+    });
+    return { duration: duration.toString(), url: link };
+  } catch (e) {
+    console.log(e);
```

Comment:
can you use `strapi.log.error` instead .. this goes into the logs on heroku, and can be seen easily